targets/TARGET_STM/TARGET_STM32WB/STM32Cube_FW/CMSIS/stm32wb5mxx.h

```diff
@@ -51,7 +51,7 @@
 #define __VTOR_PRESENT            1U /*!< Vector Table Register supported               */
 #define __NVIC_PRIO_BITS          4U /*!< STM32WBxx uses 4 Bits for the Priority Levels */
 #define __Vendor_SysTickConfig    0U /*!< Set to 1 if different SysTick Config is used  */
-#define __FPU_PRESENT             1U /*!< FPU present                                   */
+#define __FPU_PRESENT             1  /*!< FPU present                                   */ // MBED
```

Comment:
please remove redundant comment `// MBED`

Reply:
All files in TARGET_STM/TARGET_STM32xx/STM32Cube_FW
are a copy from https://github.com/STMicroelectronics/STM32CubeWB/tree/master/Drivers
This comment was just added to track the change between original repo and the copy in mbed.

Reply:
@jeromecoutant I remember this one, is it documented anywhere? It might be good to have own Readme under STM directory with section about `// MBED` comment and its usage